src/data/d2/seasonal-mod-slots.json

```diff
@@ -1,6 +1,8 @@
 {
+  "none": [],
   "forge": [65589297],
   "opulent": [1962317640, 2712224971, 1202876185],
   "outlaw": [2149155760, 13646368],
-  "undying": [1081029832]
+  "undying": [1081029832],
+  "any": [1081029832, 1202876185, 13646368, 1962317640, 2149155760, 2712224971, 65589297]
 }
```

Comment:
You cannot modify this file, you must submit a PR to https://github.com/DestinyItemManager/d2-additional-info to change the way this file is generated.

Reply:
https://github.com/DestinyItemManager/d2-additional-info/pull/84

Reply:
Thanks for the info, and for drafting the PR :) 

Should we just wait on this PR until DestinyItemManager/d2-additional-info#84 is merged then?

Reply:
merging now